hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -314,165 +314,175 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
       keys.push('Entity Type');
       values.push(summaryResponse.type);
 
-      if (summaryResponse.countStats.type === 'KEY') {
-        const keyEndpoint = `/api/v1/namespace/du?path=${path}&replica=true`;
-        const { request: metadataRequest, controller: metadataNewController } = AxiosGetHelper(keyEndpoint, cancelKeyMetadataSignal);
-        cancelKeyMetadataSignal = metadataNewController;
-        metadataRequest.then(response => {
-          keys.push('File Size');
-          values.push(byteToSize(response.data.size, 3));
-          keys.push('File Size With Replication');
-          values.push(byteToSize(response.data.sizeWithReplica, 3));
-          console.log(values);
-
-          this.setState({
-            showPanel: true,
-            panelKeys: keys,
-            panelValues: values
-          });
-        }).catch(error => {
-          this.setState({
-            isLoading: false,
-            showPanel: false
-          });
-          showDataFetchError(error.toString());
-        });
+      // If status is INITIALIZING, we cannot add entities for metadata as it will cause null failures and showing Warning message
+      // Hence we only add Entities if the status is not INITIALIZING 
+      if (summaryResponse.status ===  'INITIALIZING')
+      {
+        showDataFetchError(`The metadata is currently initializing. Please wait a moment and try again later`);
```

Comment:
For this can you display the popup message as yellow instead of green.

Reply:
@ArafatKhan2198 
Done changes suggested by you in latest commit 9f676e38d2be5834eb16fd82cde087c5e2a72ba1